examples/stable/test/property/ic_api/call/test/tests.ts

```diff
@@ -1,38 +1,74 @@
-import { expect, it, Test } from 'azle/_internal/test';
+import {
+    CallPerformFailed,
+    CallRejected,
+    expect,
+    it,
+    Test
+} from 'azle/_internal/test';
+
+import { createActor } from './test';
```

Comment:
You should get the actors just like in all of the other tests, not like you're doing it here.